pulsar-broker-auth-oidc/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationExceptionCode.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.authentication;
```

Comment:
use a unique package for this module, for example `org.apache.pulsar.broker.authentication.openid`. It's not recommended to use the same package name in different modules.

Reply:
One reason to avoid sharing package names across multiple modules is the Java Platform Module System (JPMS). A detailed answer on SO: https://stackoverflow.com/a/40635884 . 
"If you look into Java 9's class loader implementation, you can see that a package name is mapped by to a single module. Its therefore impossible two have two modules to declare ownership."

Reply:
Thanks, I was wondering if I should change this. I used the `.openid` suggestion.